src/vos/vos_layout.h

```diff
@@ -271,8 +271,13 @@ enum vos_io_stream {
 struct vos_cont_ext_df {
 	/* GC bucket extension */
 	struct vos_gc_bkt_df		ced_gc_bkt;
+	/*
+	 * Any modification involved in current target (container shard) under the global
+	 * stable epoch have already been persistently stored globally.
+	 */
+	uint64_t			ced_global_stable_epoch;
```

Comment:
Do we need to add a VOS_POOL_FEAT to control whether this is valid or not?

Reply:
We use `umem_zalloc()` to allocate the `vos_cont_ext_df`, that guarantees the unused fields (including the new added member `ced_global_stable_epoch`) for old versions will be all zero. Then when we load the local stored global stable epoch, we will find zero epoch, then will cause related integration to be started from the beginning. That is the expected behavior.